mypy/checkexpr.py

```diff
@@ -18,9 +18,8 @@
 from mypy.types import (
     Type, AnyType, CallableType, Overloaded, NoneTyp, TypeVarDef,
     TupleType, TypedDictType, Instance, TypeVarType, ErasedType, UnionType,
-    PartialType, DeletedType, UnboundType, UninhabitedType, TypeType, TypeOfAny,
-    true_only, false_only, is_named_instance, function_type, callable_type, FunctionLike,
-    get_typ_args, StarType
+    PartialType, DeletedType, UninhabitedType, TypeType, TypeOfAny, true_only,
+    false_only, is_named_instance, function_type, callable_type, FunctionLike, StarType
```

Comment:
Can you add a trailing comma? In lists like this that's how we like to format them.

Reply:
Fixed.